### PR TITLE
fix:  ui issue in login activity

### LINKED ISSUE #742: UI issue in Login Activity
**Summary:**

Mifos text is overlapping the edit text  fields in Login Activity 

**Steps to reproduce:**

Open the app

**Expected behavior:**

Normal behavior as no view overlapping.

**Observed behavior:**

Mifos text is overlapping the input fields.

**Device and Android version:**

Android Device - Redmi 4A
Android version - Nougat 7.0.1

**Screenshots:**
Screenshot_2020-02-22-04-19-40-072_org mifos mobilewallet mifospay

### PR DESCRIPTION
**Pull request by refactor-droidyy**
Fixes #742 

## Screenshots

ezgif com-video-to-gif(1)

## Description

Added scroll view as the parent view.

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by naman14** [comment]
@refactor-droidyy IMO there shouldn't be a scrollview as there's enough space to show everything. Keep constraintlayout and fix up the marging if you can

**Comment by refactor-droidyy** [comment]
@naman14 ya sure I will update it.

**Comment by refactor-droidyy** [comment]
Screenshot for Changes made -

Screenshot_2020-02-23-19-34-56-226_org mifos mobilewallet mifospay

**Comment by refactor-droidyy** [comment]
@naman14, please review

File: mifospay/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -9,8 +9,8 @@
 
     <ImageView
         android:id="@+id/image_logo"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="319dp"
```
Comment (naman14) [inline mifospay/src/main/res/layout/activity_login.xml]: Can you make these more meaningful numbers? Let's keep 300 and 160

**Comment by refactor-droidyy** [comment]
@naman14, please review

**Pull request merged**